How to Find and Use Technical Resources if You’re New to Tech:proposal.md

```diff
@@ -0,0 +1,93 @@
+
+![](https://i.imgur.com/D1WrwSL.png)
+
+Stack Overflow provides you with a large, detailed library of questions that have been answered by professionals and other seasoned programmers. Everything you find on the site is meant for you to learn from, so don't hesitate to use it.
+
+"Stack Overflow is probably a software engineer's best friend," said Bryan Wong, director of engineering at Bit Project. "Any question you have, someone else is likely to have the exact same question as you. So if I’m really stuck on a problem, Stack Overflow is the way to go."
+
+## GeeksforGeeks 
+
+Another frequently mentioned resource was [GeeksforGeeks](https://www.geeksforgeeks.org/). This site, like Stack Overflow, also provides you with lots of detailed explanations on code. But while Stack Overflow is great for looking up answers for specific questions, GeeksforGeeks helps you strengthen your understanding of the concept behind the code.
+
+![](https://i.imgur.com/j6Ke9X1.png)
+
+Many members of Bit Project — like Atul Jayaram, QA manager — pointed to this site as a great resource if you're struggling to conceptualize programming languages. Specifically, they said GeeksforGeeks was a great place to quickly learn about C++ and Python, although there are lots of other tutorials offered by the organization that focus on a range of topics like algorithims, data structures, and numerous other langauges. 
+
+So if you're looking to build a stronger understanding of why code works — not just how it works — GeeksforGeeks can be your go-to. 
+
+## W3Schools
+
+[W3Schools](https://www.w3schools.com/) is a great place to bookmark if your projects focus on web development. 
+
+![](https://i.imgur.com/JuXl3KQ.png)
+
+As you can see in the above photo, there's lots of web development languages you can learn: HTML, CSS, JavaScript, Python, ... the list goes on. 
+
+This site only offers written tutorials, but it's a good reference point when you're coding. With W3Schools, you can actually open examples in their online editor and try playing around with it to better understand how the code works. 
+
+## Youtube 
+
+Earlier, I said that search engines like Google are your best friends, but that's not 100% accurate. In reality, Google AND Youtube are your best friends. You can look for video tutorials and walkthroughts of all levels where an expert can *show* you how to code and give you an explanation. 
+
+![](https://i.imgur.com/E3zzaD6.jpg)
+
+For instance, some Bit Project members like to look up tutorials on specific coding projects; then they code along with the tutorial and add their own unique features to the project as they go. But others, like developer Kathy Liu, prefer to use Youtube tutorials about the theory of programming because they help her understand the concept of code so she can more easily apply what she's learned to future projects later on. 
```

Comment:
The last sentence is a little run-on, maybe add a comma after "theory of programming" ?